src/components/pages/Collect/Collect.js

```diff
@@ -55,6 +56,9 @@ const Collect = () => {
   const isMounted = useIsMounted()
   const handleHttpResponseError = useHttpResponseErrorHandler()
   const isReadOnlyUser = getIsUserReadOnlyForProject(currentUser, projectId)
+  const [methodsFilteredTableCellData, setMethodsFilteredTableCellData] = useState([])
+  const tableSessionStorage = JSON.parse(window.sessionStorage[`${currentUser.id}-collectTable`])
+  const [methodsFilter, setMethodsFilter] = useState(tableSessionStorage?.methodsFilter)
```

Comment:
If I comment out line 61, tests will pass, however, I need this to ensure persist works on the methodsFilter (which is why I am setting the state from` tableSessionStorage`)

I decided to implement it outside of reactTable, which does have a `columnFilters` prop, but it's mostly for a string instead of an array. The documentation does not make it clear how to implement this with different cases

Reply:
I updated it to use the existing table preferences persisted state (see most recent commit) but that might have not been the crux of it. When I loaded it up on my machine, it complained about not being able to parse undefined ( ```const tableSessionStorage = JSON.parse(window.sessionStorage[`${currentUser.id}-collectTable`])```) Tests errors also hinted at the same but less explicitly. When I fixed that and the-not-able-to-read-from-undefined errors that cascaded out from my fix, there were still issues that I didnt pursue or assess. So I am wondring if your dev environment/browser always had state that it needed, but the tests complained because they always (for this scenario at least) started from a blank state.

Reply:
I originally had it in the table, and initiated the` methods `state just before that, however, I was still seeing a bunch of failing tests. That's interesting that you're getting that undefined parsing error, as I didn't get that before!